tests/performance/specs/home-block-theme.test.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies
+ */
+const { basename, join } = require( 'path' );
+const { writeFileSync } = require( 'fs' );
+const { getResultsFilename } = require( './../utils' );
+
+/**
+ * WordPress dependencies
+ */
+import { activateTheme, createURL } from '@wordpress/e2e-test-utils';
+
+describe( 'Server Timing - Twenty Twenty Three', () => {
+	const results = {
+		wpBeforeTemplate: [],
+		wpTemplate: [],
+		wpTotal: [],
+	};
+
+	beforeAll( async () => {
+		await activateTheme( 'twentytwentythree' );
+	} );
+
+	afterAll( async () => {
+		const resultsFilename = getResultsFilename( basename( __filename, '.js' ) );
+		writeFileSync(
+			join( __dirname, resultsFilename ),
+			JSON.stringify( results, null, 2 )
+		);
+	} );
+
+	it( 'Server Timing Metrics', async () => {
+		let i = 20;
```

Comment:
Would be nice to centrally control that somewhere. At a minimum, have this defined in one place, as we should handle the number of runs consistently, not per test scenario.

Reply:
Handling this in [68e7e66](https://github.com/WordPress/wordpress-develop/pull/4139/commits/68e7e668f693df9d7bb0cab202a906acac93eb18)